src/__tests__/dpsl.test.js

```diff
@@ -510,4 +530,47 @@ describe('dpsl.diagnostics tests', () => {
       });
     });
   });
+
+
+  test('runSmartctlCheckRoutine works without parameters', (done) => {
+    const expectedRunRoutineResponse = {id: 123456};
+
+    const chrome = {
+      os: {
+        diagnostics: {
+          runSmartctlCheckRoutine:
+             (params = undefined) =>
+               Promise.resolve(expectedRunRoutineResponse),
```

Comment:
nit: i'd resolve this only when params is undefined

Reply:
Looks like we actually can check whether arguments.length === 0 to verify that there were no input params

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments

Reply:
Seems like the [jest](https://github.com/facebook/jest) framework we use for testing injects some parameters to the function call, so we can't use `arguments` in test. I sticked with `undefined` as it does the job and used `arguments` in the Diagnostics manager.